libs/webb-ui-components/src/components/ConnectWalletMobileButton/ConnectWalletMobileButton.tsx

```diff
@@ -38,24 +45,44 @@ export const ConnectWalletMobileButton = forwardRef<
         </Button>
       </BottomDialogTrigger>
       <BottomDialogPortal
-        title="Try Hubble on Desktop"
-        actionButtonsProps={actionButtonsProps}
+        title={
+          appType === 'bridge-dapp'
+            ? 'Try Hubble on Desktop'
+            : 'Switch to Desktop'
+        }
+        actionButtonsProps={
+          appType === 'bridge-dapp'
+            ? bridgeActionButtonsProps
+            : tangleActionButtonsProps
+        }
```

Comment:
You can create an additional property, such as `extraActionButtons`, and concatenate it with `actionButtonsProps`. The current condition check for the app type is not considered best practice for customization, @devpavan04.